functions-process-new-item.php

```diff
@@ -0,0 +1,25 @@
+<?php
+function alreadyInCollection(array $paints, Paints $newPaint)
+{
+    foreach ($paints as $paint) {
+        if ($newPaint->getBrandName() == $paint->getBrandname()
+            && $newPaint->getColourName() == $paint->getColourName()
+            && $newPaint->getNeedReplacing() == $paint->getNeedReplacing()) {
+            return true;
+        }
+    }
+    return false;
+}
+
+function validImageInput(string $imageFormInput)
+{
+    if (!$imageFormInput) {
```

Comment:
This could also be written as:
```
if (!$imageFormInput) {
    return true;
}

if (!filter_var($imageFormInput, FILTER_VALIDATE_URL)) {
    return false;
}

return true;
```